nexus/social_media/interface.py

```diff
@@ -0,0 +1,39 @@
+# Standard Library
+import os
+
+# Third Party Stuff
+import facebook
+from dotenv import load_dotenv
+from celery.decorators import task
+
+# Nexus Stuff
+from nexus.social_media.models import Post
+
+load_dotenv(os.path.join(os.path.dirname(os.path.dirname(__file__)), '.env'))
```

Comment:
I think better than directly loading here is to load env data into `settings` and then import it.

Reply:
Makes sense. Thanks!